### ISSUE TITLE
Add documentation for udp-option when exposing ports 

### ISSUE THREAD

**Issue by dictvm** [docker/docker.github.io#3161]
File: [engine/reference/builder.md](https://docs.docker.com/engine/reference/builder/), CC @mstanleyjones

Currently Docker Engine documentation doesn't contain references to UDP. However, Docker has support for exposing UDP ports by appending `/udp` to the port number. This should be fixed as it could mislead people to assume that UDP is either not supported or that support is spotty/incomplete. Currently you need to google "Docker udp" to learn about this functionality.

**Comment by joaofnfernandes** [comment]
Yup, you can't really find anything about UDP. There's a [reference to that here](https://docs.docker.com/engine/reference/commandline/service_create/#publish-a-port-for-tcp-only-or-udp-only), but it's hard to notice.
